### DISCUSSION TITLE
Playback App for Existing Stems?

### DISCUSSION THREAD

**Discussion by krahsloop** [Anjok07/ultimatevocalremovergui#932] [Q&A]
I recently started using UVR5 and am wondering if there exists a sound mixer app for existing stems. The subscription stem separation services like Moises often have a mixer, but of course you are restricted to using only their separation algorithm/process in their environment. I'd like to know if a tool (other than a full-on DAW) exists for simple mixing on the fly. My personal use is for music practice, to selectively mix/solo/mute stems in real-time. I envision a mobile/desktop app with a simple mixer interface similar to that of the subscription services, that allows the user to point the app to a local or cloud-based directory with existing stems. The app would populate a list of the songs in the directory, and when a song is selected, the stems would auto-populate into the mixer based on key words in the filename or just a designated naming convention i.e. Artist - Song Title (stem type). 

The two questions I have are: 1. Does this exist? 2. Would it be worthwhile to try to create it (would people find it useful)?

**Comment by ghost** [comment]
Some people are using the (old) Phase Shift player: https://dwsk.proboards.com/thread/2058/music-studio-application-ps-mixer
There's also https://www.worshipsong.com/band/get-worshipsong-band (never tried)
